app/src/main/java/com/framgia/soundclound/data/model/Track.java

```diff
@@ -0,0 +1,256 @@
+package com.framgia.soundclound.data.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Created by Sony on 1/5/2018.
+ */
+
+public class Track {
+    @SerializedName("artwork_url")
+    @Expose
+    private String mArtworkUrl;
+    @SerializedName("comment_count")
+    @Expose
+    private int mCommentCount;
+    @SerializedName("description")
+    @Expose
+    private String mDescription;
+    @SerializedName("downloadable")
+    @Expose
+    private boolean mDownloadable;
+    @SerializedName("download_count")
+    @Expose
+    private int mDownloadCount;
+    @SerializedName("download_url")
+    @Expose
+    private Object mDownloadUrl;
+    @SerializedName("duration")
+    @Expose
+    private int mDuration;
+    @SerializedName("full_duration")
+    @Expose
+    private int mFullDuration;
+    @SerializedName("genre")
+    @Expose
+    private String mGenre;
+    @SerializedName("has_downloads_left")
+    @Expose
+    private boolean mHasDownloadsLeft;
+    @SerializedName("id")
+    @Expose
+    private int mId;
+    @SerializedName("kind")
+    @Expose
+    private String mKind;
+    @SerializedName("likes_count")
+    @Expose
+    private int mLikesCount;
+    @SerializedName("permalink_url")
+    @Expose
+    private String mPermalinkUrl;
+    @SerializedName("playback_count")
+    @Expose
+    private int mPlaybackCount;
+    @SerializedName("publisher_metadata")
+    @Expose
+    private PublisherMetadata mPublisherMetadata;
+    @SerializedName("title")
+    @Expose
+    private String mTitle;
+    @SerializedName("uri")
+    @Expose
+    private String mUri;
+    @SerializedName("user_id")
+    @Expose
+    private int mUserId;
+    @SerializedName("display_date")
+    @Expose
+    private String mDisplayDate;
+    @SerializedName("user")
+    @Expose
+    private User mUser;
+
+    public Track(String title) {
+        mTitle = title;
+    }
+
+    public Track() {}
+
+    public String getArtworkUrl() {
+        return mArtworkUrl;
+    }
+
+    public void setArtworkUrl(String artworkUrl) {
+        mArtworkUrl = artworkUrl;
+    }
+
+    public int getCommentCount() {
+        return mCommentCount;
+    }
+
+    public void setCommentCount(int commentCount) {
+        mCommentCount = commentCount;
+    }
+
+    public String getDescription() {
+        return mDescription;
+    }
+
+    public void setDescription(String description) {
+        mDescription = description;
+    }
+
+    public boolean getDownloadable() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#BooleanGetMethodName